source/reflect.h2

```diff
@@ -1423,9 +145,17 @@ apply_metafunctions: (
             print( rtype );
         }
         else {
-            error( "unrecognized metafunction name: " + name );
-            error( "(temporary alpha limitation) currently the supported names are: interface, polymorphic_base, ordered, weakly_ordered, partially_ordered, copyable, basic_value, value, weakly_ordered_value, partially_ordered_value, struct, enum, flag_enum, union, print" );
-            return false;
+            (load := load_metafunction(name))
+            if load.metafunction {
+                load.metafunction( rtype );
```

Comment:
Have you thought about creating a lookup table for already loaded metafunctions?

Reply:
Yes.
But this has worked well so far.